fili-core/src/main/java/com/yahoo/bard/webservice/data/volatility/DefaultingVolatileIntervalsService.java

```diff
@@ -41,18 +47,7 @@ public DefaultingVolatileIntervalsService() {
      * @param defaultIntervalsFunction  The volatile intervals function to apply to all physical tables.
      */
     public DefaultingVolatileIntervalsService(VolatileIntervalsFunction defaultIntervalsFunction) {
-        this(new DefaultingDictionary<>(defaultIntervalsFunction));
-    }
-
-    /**
-     * Use the map of specific functions for physical tables.
-     *
-     * @param intervalsFunctions  the map of specific functions for physical tables
-     */
-    public DefaultingVolatileIntervalsService(
-                DefaultingDictionary<PhysicalTable, VolatileIntervalsFunction> intervalsFunctions
-    ) {
```

Comment:
Why are we removing this constructor outright instead of deprecating it? (call out in changelog)